docker_signal.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
The worker could be waiting for an http response.  OS signals are used to indicate a shutdown or restart is requested so it can be handled gracefully.